src/ui/base-page-storyEdit.js

```diff
@@ -385,11 +395,12 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/main/LICENSE.
             var savedStoryData = JSON.parse(window.localStorage.getItem(storyAutosaveKey));
 
             if (savedStoryData) {
+                if (!storyEdit.createStoryPromise) {
```

Comment:
If the story promise exists but still in flight, is it possible that this check may lead to two stories being created? I.e. should we check `storyEdit.createStoryPromise.disposition` as well?

Reply:
Hmm.. well I'm not really sure under what circumstance that could happen, given the timing for when the `sjrk.storyTelling.base.page.storyEdit.initializeStory` function is called. However, for arguments sake lets say that it did happen. In that case I believe the savedStoryData would be loaded into the story model. When the creation of the new story completes, it will update the story model as well, so it's probably going to update the story id. Essentially that means you'd save the story with the new id. The issue there being if there was a file already saved, it would still be named after the old story id.